frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -397,6 +398,10 @@ export function PollWorkerScreen({
   const isMarkAndPrintMode =
     machineConfig.appMode.isPrint && machineConfig.appMode.isMark;
 
+  const canSelectBallotStyle = isMarkAndPrintMode && isPollsOpen;
+  const [isHidingSelectBallotStyle, setIsHidingSelectBallotStyle] =
+    useState(false);
```

Comment:
I typically find affirmative language in booleans, e.g. `isShowingSelectBallotStyle`,  to be easier to reason about, especially when you have negations (`!isShowing` is easier to process than `!isHiding`)

Reply:
For me, in this case, it's easier to reason about as is. If it's `isShowingSelectBallotStyle` then we would default it to true. Then, `isShowingSelectBallotStyle` would be true when the "Select Ballot Style" screen is not showing on Polls Closed, or Polls Open in MarkOnly or PrintOnly mode, which is confusing to me.

Probably a better naming something but going to leave it for now to not get bogged down.